src/graphql/query/loanCardBasketed.graphql

```diff
@@ -11,12 +13,7 @@ query($id: Int!) {
 	}
 	lend {
 		loan(id: $id) {
-			id
-			loanFundraisingInfo {
-				fundedAmount
-				reservedAmount
-				isExpiringSoon
-			}
+			...loanCardFields
 		}
```

Comment:
do you actually want the full fragment here, or should the fundrasingInfo be a small fragment of its own?

Reply:
We could do that. I figured we might as well just use the whole fragment so that we don't have to update this every time we change the data fetched for the loan card.